recipe/meta.yaml

```diff
@@ -24,19 +25,21 @@ requirements:
     - setuptools
   run:
     - nltk
-    - numpy  # [not (linux and x86_64)]
-    - numpy >=1.22  # [linux and x86_64]
```

Comment:
It could be interesting to investigate why these had this condition there. Was it from upstream or is it a conda-forge only thing?

Reply:
It's not present in either conda-forge nor upstream. Seems it was [introduced here](https://github.com/AnacondaRecipes/sentence-transformers-feedstock/pull/5/commits/38a9b25aaccfee13d58a11907db49bdb75e7d4a7) because of a failing build, but I can't access the logs.

Reply:
ok, thank you for taking a look. Can you try to run the tests on linux-64 with numpy <1.22? To do that, you can download the built package and run `conda build -t <package> --extra-deps numpy=1.21` for example.

Reply:
I just finished running the full build including tests on a linux-64 machine, and they pass on linux-64 with `numpy 1.21`.